packages/js-sdk/tests/sandbox/commands/sendStdin.test.ts

```diff
@@ -37,6 +37,8 @@ sandboxTest('send special characters to stdin', async ({ sandbox }) => {
 
   await sandbox.commands.sendStdin(cmd.pid, text)
 
+  await wait(5_000)
```

Comment:
This isn't probably needed, should be solved by the loop bellow, feel free to increase the loop range